dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM php:7-cli
+
+RUN apt-get update 
+RUN apt-get install -y curl zip
+
+RUN curl -sS https://getcomposer.org/installer | php
+RUN mv composer.phar /usr/local/bin/composer
+RUN composer global require "phpunit/phpunit"
```

Comment:
phpunit should already be a dev-dependency of the project, so in theory you should be able to run `./vendor/bin/phpunit`

Reply:
Once I did this the tests passed 🤔